packages/synapse-react-client/src/components/TermsAndConditions/TermsAndConditionsItem.tsx

```diff
@@ -19,6 +28,43 @@ const TermsAndConditionsItem: React.FunctionComponent<
   TermsAndConditionsItemProps
 > = props => {
   const { id, item, enabled, checked, onChange } = props
+  const { iconFileHandleId, label, description } = item
+  const [iconFileResult, setIconFileResult] = useState<FileResult | undefined>(
+    undefined,
+  )
+  const [iconFileContent, setIconFileContent] = useState<string | undefined>(
+    undefined,
+  )
+
+  useEffect(() => {
+    if (iconFileHandleId) {
+      SynapseClient.getFiles({
+        requestedFiles: [
+          {
+            associateObjectId: termsAndConditionsTableID,
+            associateObjectType: FileHandleAssociateType.TableEntity,
+            fileHandleId: iconFileHandleId,
+          },
+        ],
+        includeFileHandles: true,
+        includePreSignedURLs: true,
+        includePreviewPreSignedURLs: false,
+      }).then(fileResults => {
+        setIconFileResult(fileResults.requestedFiles[0])
+      })
+    }
+  }, [iconFileHandleId])
+
+  useEffect(() => {
```

Comment:
Once we have the file result, read the file content (svg icon data), to be injected into the html